[DOCS] Create New Content for Column Descriptive Metrics in GX Cloud (#8814)
great-expectations/great_expectations: docs/docusaurus/docs/cloud/data_assets/manage_data_assets.md

> Data Asset metrics provide you with insight into the data you can use with your Expectations and Validations. 

Reviewer: ```suggestion
Data Asset metrics provide you with insight into the data you can use for your data validations. 
```

Author: @talagluck your change has been implemented
